projects/control-service/projects/helm_charts/pipelines-control-service/values.yaml

```diff
@@ -151,7 +151,7 @@ deploymentGitPassword: ""
 # Credentials with read and write access to the Git repository.
 uploadGitReadWriteUsername: ""
 uploadGitReadWritePassword: ""
-
+uploadValidationFileTypesAllowList: ""
```

Comment:
We use values.yaml as an interface so all properties need be documented. 

You can steal the doc I did in application.properties in https://github.com/vmware/versatile-data-kit/pull/1268